etsin_finder/frontend/js/components/qvain/general/qvainTooltip.jsx

```diff
@@ -0,0 +1,35 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import translate from 'counterpart'
+import Tooltip from './tooltip'
+import { HelpIcon } from './form'
+
+class QvainTooltip extends PureComponent {
+  static propTypes = {
+    tooltipAriaLabel: PropTypes.string.isRequired,
+    TooltipContent: PropTypes.elementType.isRequired,
```

Comment:
Maybe use same capitalization for both props?
```suggestion
    tooltipAriaLabel: PropTypes.string.isRequired,
    tooltipContent: PropTypes.elementType.isRequired,
```

Reply:
changed into lower-case notation. Needed extra line to assign tooltipContent component to TooltipContent. React prohibits rendering components with lower case notation.